multiqc/modules/peddy/peddy.py

```diff
@@ -54,6 +58,16 @@ def __init__(self):
                             self.peddy_data[s_name].update(parsed_data[s_name])
                         except KeyError:
                             self.peddy_data[s_name] = parsed_data[s_name]
+        # parse background PCA JSON file
+        for pattern in ['background_pca']:
+            sp_key = 'peddy/{}'.format(pattern)
+            for f in self.find_log_files(sp_key):
```

Comment:
Am I right in thinking that these files should always be identical? Can we parse just the first one found if so?

Reply:
Will do-- was thinking there were other json files to parse when I was writing it, but turns out not.